src/Sarif.Converters/StaticDriverVerifierConverter.cs

```diff
@@ -225,6 +225,11 @@ private void ProcessLine(string logFileLine, ref int nestingLevel, Result result
                     }
                 }
 
+                if (threadFlowLocation.NestingLevel == 0)
```

Comment:
>NestingLevel [](http://example.com/codeflow?start=39&length=12)

why is this correct?

how do we distinguish between not present vs. an actual nesting level of 0?

Reply:
you are correct, this is code for converting StaticDriverVerifier result to SARIF, the NestingLevel here is converted so if it is 0 then it means 0, not unknown. I have reverted this change, and fixed the test expected outputs to have this property.